src/search/utils/QueryStringGenerator.ts

```diff
@@ -56,16 +48,10 @@ const createValueString = (
   id: string | undefined
 ) => {
   if (term === undefined) {
-    return '';
-  }
-  if (term === 'ec') {
-    if (id) {
-      return id;
-    }
-    throw new Error('Value not provided in query');
+    throw new Error('term is undefined');
   }
-  if (['organism', 'taxonomy', 'host'].includes(term) && id) {
-    return id;
+  if (id) {
+    return `"${id}"`;
```

Comment:
I'm wondering if it would be worth doing the same with `stringValue` a few lines below (or maybe it also needs the prefix!?) and wrapping it in `"` to escape special characters. Check with Jie

Reply:
Yeah, will take that on in https://www.ebi.ac.uk/panda/jira/browse/TRM-22877